test/cmd/apply.sh

```diff
@@ -444,6 +444,16 @@ run_kubectl_server_side_apply_tests() {
 }
 __EOF__
 
+  # Ensure the API server has recognized and started serving the associated CR API
+  local tries=5
+  for i in $(seq 1 $tries); do
+      if kubectl "${kube_flags_with_token[@]}" api-resources --api-group 'mygroup.example.com' -oname | grep -q 'resources.mygroup.example.com'; then
+          break
+      fi
+      echo "${i}: Waiting for CR API to be available"
+      sleep $((i-1))
```

Comment:
I may be missing something but why the changing sleep time? Is it intended to be a rudimentary backoff?

Reply:
no special reason, just follow the existing practice https://github.com/kubernetes/kubernetes/blob/c3816157f97cc068600b0dbebdbe80733ee5c3eb/hack/lib/test.sh#L82-L93

Reply:
we could fix the sleep time if that is preferred

Reply:
@knight42 I think it's okay, just wanted to know the motivation :)

Reply:
this is similar, used in other places, and is easier to read:
```suggestion
      sleep "${i}"
```